style.css

```diff
@@ -0,0 +1,59 @@
+.main_content {
+    text-align: center;
+    background-color: gainsboro;
+    border-radius: 10px;
+}
+
+.main_header {
+    margin: 20px 0px 0px 0px;
```

Comment:
Zero value property cannot have a unit. Please remove the unit from all zero values. It should "margin: 20px 0 0 0;"